power_grid_model_io_native_c/power_grid_model_io_native/CMakeLists.txt

```diff
@@ -2,8 +2,10 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-add_library(power_grid_model_io_native INTERFACE)
+add_library(power_grid_model_io_native STATIC) #static
 
-target_link_libraries(power_grid_model_io_native INTERFACE power_grid_model)
+set_target_properties(power_grid_model_io_native PROPERTIES LINKER_LANGUAGE CXX)
```

Comment:
this should not be necessary if you configure the CMake project in the root correctly